src/parcsr_ls/par_gauss_elim.c

```diff
@@ -338,7 +338,7 @@ hypre_GaussElimSetup(hypre_ParAMGData *amg_data,
     *  Factorization phase
     *-----------------------------------------------------------------*/
 
-#if defined(HYPRE_USING_GPU)
```

Comment:
I guess no sycl support for this yet.

Reply:
Yes, sorry, my bad!

Reply:
Correct, this functionality is brand new from a recent PR from Victor, so I'll look at that for a future PR.